papis/utils.py

```diff
@@ -137,15 +131,17 @@ def general_open(file_name: str,
     import shutil
     if not shutil.which(cmd[0]):
         raise FileNotFoundError(
-            "[Errno 2] No such file or directory: '{}'".format(opener))
+            "[Errno 2] No such file or directory: '{}'".format(cmd[0]))
```

Comment:
Error string should be changed to something like `Command not found : {cmd[0]}`

Reply:
Yeah, good idea! 

I'll leave it as a `FileNotFoundError` though, because that's what `subprocess.call` throws as well if the command is not found, e.g.
```
>>> subprocess.call(["non-existent-command"])
...
FileNotFoundError: [Errno 2] No such file or directory: 'non-existent-command'
```